src/mongo/delegates/Tables.class.php

```diff
@@ -362,8 +371,9 @@ public function deleteTableRowsByTableId($tableId) {
             return;
         }
 
+        $query = array("_id.type"=>$tableId);
         $this->config->getCollectionForTable($this->storeName, $tableId)
-            ->remove(array("_id.type"=>$tableId), array('fsync'=>true));
+            ->deleteMany($query);
```

Comment:
I've tracked the result of this and logged out if there was an issue acknowledging the delete/update.